test/literate.litcoffee

```diff
@@ -50,8 +50,3 @@ and unordered lists, are fine:
   * A
   
   * List
-
```

Comment:
Please restore this test. I know it’s broken. It will get fixed separately. You can keep it deleted or commented out in your local copy, but don’t commit that change.